app/graphql/types/concerns/system_profiles.rb

```diff
@@ -15,6 +15,12 @@ def test_result_profiles(policy_id: nil)
       scope_profiles(object.test_result_profiles, policy_id)
     end
 
+    def policies(policy_id: nil)
+      context_parent
+      policy_profiles = object.assigned_profiles.external(false)
+      scope_profiles(policy_profiles, policy_id)
+    end
```

Comment:
Same comment here wrt naming. Let's be as explicit as possible. Naming it `policies` only increases the confusion surrounding an already confusing topic. There is really no detriment to having descriptive attribute names in our APIs. `assigned_internal_profiles` or similar would communicate better what is being returned.

Reply:
Naming is wrt `Profile.policy`. The name "policies" is deliberate, in a sense that we can change the underlying type keeping the subset of `Profile` type policy-related fields.

`assigned_internal_profiles` would not be accurate. These has to return external profiles/policies until we drop the external support completely. Only "profiles" is accurate from the name. There is no such thing as assigned profile, but an assigned policy. "Internal" adds more confusion to it TBH, as everything would be/is internal.

Let me think of some other name... We need remember that we are in the v1 sphere (even if the GQL isn't versioned) where the real policy does not exist.

Reply:
Think about it on the abstract level where the first policy profile represents *the* policy.

Reply:
I understand what the idea is here, I just don't think the naming accurately reflects what's being returned. One day, we may introduce actual policies to the GraphQL API, and we'll be in a pickle for what to name them. Since we already have the concept of policies on a profile in our GraphQL API, I guess it's okay here since it matches.

Reply:
Introduction of actual policies on the same field would have very minimal impact. Consumers, including our fronted, would not need many changes. The choice for this name is by design, specifically for this situation.

Reply:
My alternative name would be `policy_profiles`.

Reply:
Ok, that's a fair point. I guess the only thing that will change will be the `id` returned with the policy if we ever actually start returning policy objects.